packages/ember-glimmer/tests/integration/custom-component-manager-test.js

```diff
@@ -0,0 +1,27 @@
+import { moduleFor, RenderingTest } from '../utils/test-case';
+import TestComponentManager from './helpers/test-component-manager';
```

Comment:
I'd prefer to just implement a super simple manager in this file directly, for now we don't have a need for it across many files...